_posts/2022-12-21-rieck22a.md

```diff
@@ -13,7 +13,7 @@ lastpage: xxiii
 page: i-xxiii
 order: 0
 cycles: false
-bibtex_author: Rieck, Bastian and Pascanu, Razvan and Du, Yuanqi and St"{a}rk, Hannes
+bibtex_author: Rieck, Bastian and Pascanu, Razvan and Du, Yuanqi and Stärk, Hannes
```

Comment:
```suggestion
bibtex_author: Rieck, Bastian and Pascanu, Razvan and Du, Yuanqi and St\"{a}rk, Hannes
```

Reply:
Your instinct was right, here we use the latex style (for creating the bibtex, put the additional fix in).